lte/gateway/c/session_manager/SessionManagerServer.cpp

```diff
@@ -76,8 +76,25 @@ AmfPduSessionSmContextAsyncService::AmfPduSessionSmContextAsyncService(
     : AsyncService(std::move(cq)), handler_(std::move(handler)) {}
 
 void AmfPduSessionSmContextAsyncService::init_call_data() {
-  new SetAmfSessionContextCallData(cq_.get(), *this, *handler_);
   MLOG(MINFO) << "Initializing new call data for SetAmfSessionContext";
+  new SetAmfSessionContextCallData(cq_.get(), *this, *handler_);
+  MLOG(MINFO) << "Initializing new call data for SetSmfNotification";
+  new SetSmfNotificationCallData(cq_.get(), *this, *handler_);
+}
+
+/*Landing object invocation object call for 5G*/
+SetInterfaceForUserPlaneAsyncService::SetInterfaceForUserPlaneAsyncService(
```

Comment:
what is this function?

Reply:
If u look latest  session_manager proto @1021 , these are newly added GRPC services provided by SMF for UPF client to send and these are part of new MVC code.  Just to add more, with this converged code all  above UPF 3 GRPC service request are handled by SMF with newly added UPF thread.   Let me know if you are looking any more detilas on these.